lms/envs/common.py

```diff
@@ -1328,6 +1328,20 @@
         ],
         'output_filename': 'css/lms-style-xmodule-annotations.css',
     },
+    'style-certificates': {
+        'source_filenames': [
+            'sass/certificates/main-ltr.css',
```

Comment:
@talbs Why shouldn't the sass get compiled, anyhow? I'm confused on that matter. If you don't compile the assets, the view will not render correctly. And you certainly need to make sure compiling is consistent in order for bok_choy tests to pass.